### DISCUSSION TITLE
Script transmission-post-start never running

### DISCUSSION THREAD

**Discussion by kkirby** [haugene/docker-transmission-openvpn#2347] [Q&A]
I have a script `/scripts/transmission-post-start.sh` that is never ran post start. I can run the script directly inside the container without any issues.

Is there something I'm missing? I have `#!/bin/bash` and the file's permissions are `775`. Ideally, I'd want this script to run whenever the VPN connects.

**Comment by pkishino** [comment]
You followed this, right?
http://haugene.github.io/docker-transmission-openvpn/config-options/

**Reply by kkirby** [reply]
Correct.

**Reply by kkirby** [reply]
I did a little digging. It looks like script does get executed when the container starts up. But if the VPN goes down, and comes back up, it does not get executed.
